app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -149,7 +149,12 @@ public void listenForChangeInSettings() {
 
     public void setRetryTimer() {
         if (CollectionServiceStarter.isBTWixel(getApplicationContext()) || CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
-            long retry_in = (1000 * 65);
+            long retry_in;
+            if(CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
+                retry_in = (1000 * 25);
+            } else {
+                retry_in = (1000*65);
```

Comment:
Yeah, fair enough.  I do kinda like the readability of (1000 * X) though.
There is a lot of this sort of thing throughout xDrip.  Normally I just put
the final number in and forget about the calculation, but I am pedantic
about not using CPU cycles for anything but what they are needed for ;)
Fix it up if you like.
Cheers

On Wed, Nov 11, 2015 at 6:40 AM, Matthias Granberry <
notifications@github.com> wrote:

> In
> app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/153#discussion_r44454252>
> :
>
> > @@ -149,7 +149,12 @@ public void listenForChangeInSettings() {
> >
> >      public void setRetryTimer() {
> >          if (CollectionServiceStarter.isBTWixel(getApplicationContext()) || CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
> > -            long retry_in = (1000 * 65);
> > +            long retry_in;
> > +            if(CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
> > +                retry_in = (1000 * 25);
> > +            } else {
> > +                retry_in = (1000*65);
>
> [image: Codacy]
> <https://camo.githubusercontent.com/b1b1514cd8a7d7c9c9fa32f769284569ffee2882/68747470733a2f2f7777772e636f646163792e636f6d2f76657273696f6e65642f696d616765732f66617669636f6e2e706e67>
> Issue found: Useless parentheses.
> <https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1301958408/issues/source?bid=2735053&fileBranchId=2947766#l156>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/153/files#r44454252>
> .
>



-- 
John Stevens
"You are how you live, not what you have."